common/services/population.js

```diff
@@ -51,6 +51,25 @@ class PopulationService extends BaseService {
       .then(response => response.data)
   }
 
+  populate(data) {
+    const { location, date } = data
+
+    if (!location) {
+      return Promise.reject(new Error(noLocationIdMessage))
+    }
+
+    if (!date) {
+      return Promise.reject(new Error(noDateMessage))
+    }
+
+    return this.apiClient
+      .findAll('populate', {
```

Comment:
I think on reflection, the previous implementation was a little easier to understand what was going on.

However, if you don't want to spend any time reverting it, I understand.